test/physics/material/Material.test.cc

```diff
@@ -103,14 +103,16 @@ class MaterialTest : public celeritas::Test
             {53, AmuMass{126.90447}, "I"},
         };
         inp.materials = {
-            {0.05 * constants::na_avogadro, // XXX fix number density
+            {2.948915064677e+22,
              293.0,
              MatterState::solid,
              {{ElementDefId{2}, 0.5}, {ElementDefId{3}, 0.5}},
              "NaI"},
-            {0.0, 0.0, MatterState::unspecified, {}, "hard vacuum"},
-            {1e-5 * constants::na_avogadro, // XXX fix number density
-             100.0,
+
+            {0, 0, MatterState::unspecified, {}, "hard vacuum"},
+
+            {4.1822910994920445e+22,
+             293.0,
```

Comment:
This temperature accidentally got changed (and in the test of the temperature below).

Also, does hydrogen really have 2x the atomic number density of sodium iodide??

Reply:
Only at melting point. I mistakenly left old test numbers for hydrogen. Fixed now.